### PR TITLE
added GCI students from Belgaum, India

### PR DESCRIPTION
**Pull request by wavicles**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://wavicles.github.io/gci15.fossasia.org

**Comment by mananwason** [comment]
@wavicles Please remove the merge commit and squash the other 2 commits. Thanks! :smile:

File: _data/students.yml [hunk 1]
```diff
@@ -619,6 +619,64 @@
...
+  bio: It is always the adventurous who accomplish great things!
+
+- name: Anushri
+  image: anushri.jpg
+  twitter: Uanushri
+  bio: In search of purpose of existance
+
+- name: Akshata
+  image: akshata.jpg
+  twitter:swararevankar
```
Comment (jigyasa-grover) [inline _data/students.yml]: **Build failing**
Include a space after `twitter:` and before  `swararevankar`.

**Comment by wavicles** [comment]
Thanks @jig08 @mananwason 
How do i remove merge commit?
I am in learning mode :(

**Comment by wavicles** [comment]
Should I close this pull request and do it again?

**Comment by mananwason** [comment]
@wavicles I am sorry, couldn't respond sooner. Please don't merge PR's with bad commits. It Degrades the codebase. Thanks!

**Pull request merged**